devtools/cli/src/main/java/io/quarkus/cli/Encryptor.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.cli;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.concurrent.Callable;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import io.quarkus.cli.common.OutputOptionMixin;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "encryptor", header = "Encrypt Secrets in AES/GCM/NoPadding")
```

Comment:
```suggestion
@Command(name = "encryptor", header = "Encrypt Secrets using AES/GCM/NoPadding algorithm by default")
```

Reply:
Or something similar, as users would be able to choose a different algorithm if they prefer.